src/cdk/testing/testbed/testbed-harness-environment.ts

```diff
@@ -43,33 +43,33 @@ export class TestbedHarnessEnvironment extends HarnessEnvironment<Element> {
   static async harnessForFixture<T extends ComponentHarness>(
       fixture: ComponentFixture<unknown>, harnessType: ComponentHarnessConstructor<T>): Promise<T> {
     const environment = new TestbedHarnessEnvironment(fixture.nativeElement, fixture);
-    await environment._stabilize();
+    await environment.forceStabilize();
     return environment.createComponentHarness(harnessType, fixture.nativeElement);
   }
 
+  async forceStabilize(): Promise<void> {
+    if (this._destroyed) {
+      throw Error('Harness is attempting to use a fixture that has already been destroyed.');
```

Comment:
nit: Something that I just thought about that I should've included when I added the error: it might be useful to log out the name of the harness is here. You should be able to get it through `this.harnessType.name`.

Reply:
I think there's not actually a good way to get the harnessType here since this is in `TestbedHarnessEnvironment`. I think it should be ok though, the name should show up in the stack trace.